### DISCUSSION TITLE
Shopping Cart

### DISCUSSION THREAD

**Discussion by Shady25-max** [community/community#128243] [Programming Help]
### Select Topic Area

Question

### Body

how to come up with a shopping cart using html, css and javascript

**Comment by ebndev** [comment]
Hi @Shady25-max, thanks for being a part of the GitHub Community!

[Use Shopping Cart](https://useshoppingcart.com/) is a great open-source solution using React and Stripe if you don't want to want to code everything from scratch! It would be hard to due it with pure HTML, CSS, and JavaScript since you'll likely have to connect a payment platform's API.

**Comment by LivexTwin** [comment]
Do you want to create the look of a shopping cart or you want to add payment functionality?
